src/scielo/bin/xml/prodtools/config/config.py

```diff
@@ -35,19 +35,22 @@ def __init__(self, filename=None):
             self.filename = None
         self.load()
 
-    def load(self):
-        self._data = {}
+    def read_files(self):
         content_files = ''
         for item in ['scielo_env.ini', 'scielo_collection.ini']:
-            if os.path.isfile(os.path.join(BIN_PATH, item)):
-                content_files += "\n" + fs_utils.read_file(
-                    os.path.join(BIN_PATH, item))
+            file_path = os.path.join(BIN_PATH, item)
+            content_files += (fs_utils.read_file(file_path) or '') + "\n"
 
         if self.filename is not None:
             coding = 'utf-8'
             if self.filename.endswith('scielo_paths.ini'):
                 coding = 'iso-8859-1'
             content_files += fs_utils.read_file(self.filename, coding)
+        return content_files
+
+    def load(self):
```

Comment:
@robertatakenaka, segue uma sugestão que eliminaria os métodos `read_files` e `load`: https://gist.github.com/gustavofonseca/a5884ecde5e034bff979f6025b09c0ac

Reply:
@gustavofonseca mas teria que mexer em outras coisas ainda, pois o scielo_paths.ini é originalmente um arquivo para uso do title manager, converter etc. Então tem umas particularidades que são tratadas pelo read_files.

Reply:
Tudo bem, fica como sugestão de melhoria futura.

Reply:
As peculiaridades que vc diz são `SGML Parser Program Directory=c:\programas\scielo\bin\sgmlpars\,required`? Tipo, teria que eliminar o `,required`?

Reply:
A segunda versão limpa sufixo `,required`, se esta for a única particularidade.